docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -517,29 +517,29 @@ public class PersistenceAndQuarkusConfigTest {
 <1> This tells JUnit that the {project-name} deployment should fail with a specific exception
 
 
-== Native Image Support
+== Native Executable Support
 
-There Quarkus provides a lot of build items that control aspects of the native image build. This allows for extensions
+There Quarkus provides a lot of build items that control aspects of the native executable build. This allows for extensions
 to programmatically perform tasks such as registering classes for reflection or adding static resources to the native
 image. Some of these build items are listed below:
 
 `io.quarkus.deployment.builditem.substrate.SubstrateResourceBuildItem`::
- Includes static resources into the native image.
+ Includes static resources into the native executable.
 
 `io.quarkus.deployment.builditem.substrate.RuntimeReinitializedClassBuildItem`::
 A class that will be reinitialized at runtime by Substrate. This will result in the static initializer running twice.
 
 `io.quarkus.deployment.builditem.substrate.SubstrateSystemPropertyBuildItem`::
-A system property that will be set at native image build time.
+A system property that will be set at executable image build time.
```

Comment:
Why do you use `executable image` here and below?

I could see how we could use `native image` here instead of `native executable` but `executable image` looks weird.